src/ai/backend/manager/cli/__main__.py

```diff
@@ -170,6 +181,373 @@ def clear_history(cli_ctx: CLIContext, retention, vacuum_full) -> None:
+    config_parser["handler_console"]["formatter"] = handler_console_formatter
+
+    with open("alembic.ini", 'w') as f:
+        print("\nDump to alembic.ini\n")
+        config_parser.write(f)
+
+    # Dump etcd config json
+    with open("config/sample.etcd.config.json") as f:
+        config_json: dict = json.load(f)
+
+    try:
+        if config_json.get("redis") is None:
+            raise KeyError
+        elif type(config_json.get("redis")) != dict:
+            raise TypeError
+        redis_config: dict = dict(config_json["redis"])
+        while True:
+            redis_host_str, redis_port_str = str(redis_config.get("addr")).split(":")
+            redis_host = ask_host("Redis host: ", str(redis_host_str))
+            if type(redis_port_str) != str:
+                raise TypeError
+            redis_port = ask_number("Redis port: ", int(redis_port_str), 1, 65535)
+            redis_password = ask_string("Redis password", use_default=False)
+            if redis_password:
+                redis_client = aioredis.Redis(
+                    host=redis_host,
+                    port=redis_port,
+                    password=redis_password)
+            else:
+                redis_client = aioredis.Redis(host=redis_host, port=redis_port)
+
+            try:
+                loop = asyncio.get_event_loop()
+                coroutine = redis_client.get("")
+                loop.run_until_complete(coroutine)
+                coroutine = redis_client.close()
+                loop.run_until_complete(coroutine)
+                config_json["redis"]["addr"] = f"{redis_host}:{redis_port}"
+                config_json["redis"]["password"] = redis_password
+                break
+            except (aioredis.exceptions.ConnectionError, aioredis.exceptions.BusyLoadingError):
+                print("Cannot connect to etcd. Please input etcd information again.")
+
+        while True:
+            timezone = input("System timezone: ")
+            try:
+                _ = ZoneInfo(timezone)
+                config_json["system"]["timezone"] = timezone
+                break
+            except (ValueError, ZoneInfoNotFoundError):
+                print('Please input correct timezone.')
+    except ValueError:
+        raise ValueError
+
+    with open("dev.etcd.config.json", "w") as f:
+        print("\nDump to dev.etcd.config.json\n")
+        json.dump(config_json, f, indent=4)
+
+    print("Complete configure backend.ai manager. "
+          "If you want to control more value, edit following files.\n")
+    print("manager.toml : etcd, database, manager configuration, logging options and so on.")
+    print("alembic.ini : option about alembic")
+    print("dev.etcd.config.json : etcd options like timezone, host, port and so on.")
+
+
+def check_etcd_health(host: str, port: int):
+    try:
+        etcd_client = etcd3.Etcd3Client(host=host, port=port)
+        etcd_client.close()
```

Comment:
In production, the etcd may be configured with user account with explicit credentials. (Note: configuring user credentials in etcd is *not* our scope here.)